Frontend/src/app/components/research-lab/signal-report/signal-report.component.html

```
@@ -306,11 +335,30 @@ <h4 class="table-title mb-3">
+                <div class="sharpe-ci-label">95% CI on combined OOS Sharpe</div>
+                <div class="sharpe-ci-value">
+                  [{{ ci.ciLower | number: '1.2-2' }}, {{ ci.ciUpper | number: '1.2-2' }}]
+                </div>
+                <div class="sharpe-ci-se">SE = {{ ci.se | number: '1.3-3' }}</div>
+                <div class="sharpe-ci-neff">N<sub>eff</sub> = {{ ci.nEffUsed | number: '1.0-0' }}</div>
+              </div>
+              <div class="sharpe-ci-note">
+                <i class="pi pi-info-circle"></i>
+                <span>
+                  Lo (2002) interval, autocorrelation-adjusted via N<sub>eff</sub>.
+                  @if (ci.ciLower < 0 && ci.ciUpper > 0) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)